Makefile

```diff
@@ -197,6 +197,9 @@ push-openshift-dev: set-openshift-image-params openshift-user image
 
 .PHONY: do-deploy-openshift-dev
 do-deploy-openshift-dev:
+	@echo "Deploying cert-manager"
+	oc apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
```

Comment:
This installs cert-manager from upstream, which is also what the e2e tests do, that's why I kept the same thing here. But it turns out that OCP ships cert-manager-operator in the openshift-marketplace:
```
 oc describe packagemanifests cert-manager-operator -nopenshift-marketplace
```
It might be possible to install that operator instead.

Reply:
This is fine for development though. Ultimately, we could even use OCP's own cert-rotation controllers... and when we package this for OLM, the OLM has its own way of providing certs too. So, let's keep this as-is and change later if needed.